graphqlapi/src/main/java/io/stargate/graphql/schema/CustomScalar.java

```diff
@@ -171,6 +170,34 @@ private static long parseLong(Object v) {
         String.format("Expected string for bigint scalar, obtained %s", v.getClass().getName()));
   }
 
+  /** Support uuid literals and now() function */
+  private static java.util.UUID parseTimeUuid(Object v) {
+    if (!(v instanceof String)) {
+      throw new IllegalArgumentException(
+          String.format("Expected string value for TimeUuid, obtained: %s", v));
+    }
+
+    String stringValue = (String) v;
+    if (stringValue.equals("now()")) {
+      return Uuids.timeBased();
+    }
+    return java.util.UUID.fromString(stringValue);
+  }
+
+  /** Support uuid literals and uuid() function */
+  private static java.util.UUID parseUuid(Object v) {
+    if (!(v instanceof String)) {
+      throw new IllegalArgumentException(
+          String.format("Expected string value for Uuid, obtained: %s", v));
+    }
+
+    String stringValue = (String) v;
+    if (stringValue.equals("uuid()")) {
+      return Uuids.random();
+    }
+    return java.util.UUID.fromString(stringValue);
+  }
+
```

Comment:
When you rebase this will be in `StringCoercing`.

Reply:
Rebased and moved :) 👍